cmd/mdatagen/templates/documentation.md.tmpl

```diff
@@ -83,11 +83,19 @@ metrics:
 
 ## Resource Attributes
 
+| Name | Description | Type | Enabled |
+| ---- | ----------- | ---- | ------- |
+{{- range $attributeName, $attributeInfo := .ResourceAttributes }}
+| {{ $attributeName }} | {{ $attributeInfo.Description }} | {{ $attributeInfo.Type }} | {{ $attributeInfo.Enabled }} |
+{{- end }}
+{{- end }}
+
+## Metric attributes
```

Comment:
Please remove this section. Metrics attributes are rendered separately for each metric